bin/pipelineTiming.sh

```diff
@@ -209,99 +209,99 @@ rsync -av ${sourceServerFQDN}:"${SCR_ROOT_DIR}/logs/Timestamp/*.csv" "${projectS
-            touch "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.failed"
-            echo -e "Dear GCC helpdesk,\n\nPlease check if there is somethink wrong with the pipeline.\nThe pipeline for project ${project} is not finished after 6h.\n\nKind regards\nGCC" > "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.failed"
-            log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "pipeline.started file is OLDER than 6 hour for project ${project}"
-            continue
-        fi
-    fi 
-    
-    ## step 3, check the copyProjectDataToPrm
-    if [ -e "${logsDir}/${project}/${run}.copyProjectDataToPrm.finished" ]
-    then
-        echo -e "copyProjectDataToPrm is finished for project ${project}" >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
-        touch "${TMP_ROOT_DIR}/logs/${project}/${run}.copyProjectDataToPrmTiming.finished"
-        log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "copyProjectDataToPrm finished for project ${project}"
-    else
-        timeStampCopyProjectDataToPrm=$(find "${logsDir}/${project}/" -type f -mmin +300 -iname "${run}.copyProjectDataToPrm.started")
-        if [[ -z "${timeStampCopyProjectDataToPrm}" ]]
-        then
-            echo -e "copyProjectDataToPrm has not started jet, or is running" >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
-            log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "copyProjectDataToPrm has not started yet or is running for project ${project}"
-            continue
-        else
-            echo -e "copyProjectDataToPrm.started file is OLDER than 5 hour.\n" \ 
-            "time ${run}.copyProjectDataToPrm.started was last modified:\n" \
-            $(stat -c %y "${TMP_ROOT_DIR}/logs/${project}/${run}.copyProjectDataToPrm.started") >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
-            touch "${TMP_ROOT_DIR}/logs/${project}/${run}.copyProjectDataToPrmTiming.failed"
-            echo -e "Dear GCC helpdesk,\n\nPlease check if there is somethink wrong with the pipeline.\nThe copyProjectDataToPrm has started but is not finished after 6h for project ${project}.\n\nKind regards\nGCC" > "${TMP_ROOT_DIR}/logs/${project}/${run}.copyProjectDataToPrmTiming.failed"
-            log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "copyProjectDataToPrm.started file OLDER than 5 hour for project ${project}"
-            continue
-        fi
-    fi
-
-    echo -e "The complete pipeline is finished for project ${project}" >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
-    log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "The complete pipeline is finished for project ${project}"
-    
-    ssh ${sourceServerFQDN} "mv ${SCR_ROOT_DIR}/logs/Timestamp/${project}.csv ${SCR_ROOT_DIR}/logs/Timestamp/archive/"
-    mv "${projectStartDir}/${project}.csv" "${projectStartDir}/archive/"
+	if [[ ! -e "${projectSheet}" ]]
+	then
+		continue
+	fi
+	project=$(basename "${projectSheet}" .csv)
```

Comment:
no space between "${projectSheet}" and .csv?

Reply:
nee, projectsheet heeft als excetie .csv. met die .csv erachter haal je dat er af.